demisto_sdk/commands/validate/validators/IN_validators/IN144_is_rep_command_contain_is_array_argument.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import annotations
+
+from typing import Dict, Iterable, List
+
+from demisto_sdk.commands.common.constants import (
+    BANG_COMMAND_ARGS_MAPPING_DICT,
+    BANG_COMMAND_NAMES,
+)
+from demisto_sdk.commands.content_graph.objects.integration import Command, Integration
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Integration
+
+
+class IsRepCommandContainIsArrayArgumentValidator(BaseValidator[ContentTypes]):
+    error_code = "IN144"
```

Comment:
this isnt a new validation right? Just porting this over? seems like a strange validation

Reply:
Not a new one, but it is a relevant one.. we should support isArray for all rep commands.